app/controllers/api_controller/chargeback.rb

```diff
@@ -0,0 +1,45 @@
+class ApiController
+  module Chargeback
+    # Request
+    # {
+    #   "action":"assign",
+    #
+    #   "category": "tenant",
+    #   "c_id": 1
+    # }
+    def chargebacks_assign_resource(object, _type, _id, data)
+      klass = Object.const_get(data['category'].camelcase)
```

Comment:
You may want to use `.constantize` here.

Do you think it would make sense to whitelist the possible values for category here?